web/cobrands/shropshire/assets.js

```diff
@@ -90,7 +88,9 @@ fixmystreet.assets.add(defaults, {
             TYPENAME: "Traffic_Signal_Areas"
         }
     },
-    feature_code: 'FEATURE_ID',
+    attributes: {
+        central_asset_id: 'ASSET_ID',
```

Comment:
This layer *does* have SITE_CODE, so any reason not to use it? (I'm guessing maybe these are not the same site codes as we're expecting, but thought I should check)

Reply:
Wasn't sure - I thought sticking to the same asset id as provided by the default would be fine, but don't know...do you think I should?

Reply:
I'd hoped you would have checked my guess about whether they weren't the same site codes or not :) I've done that now, they don't look the same at all, so no, don't use them.